docs/dask_cudf/source/best_practices.rst

```diff
@@ -297,11 +297,16 @@ bottleneck is typically device-to-host memory spilling.
 Although every workflow is different, the following guidelines
 are often recommended:
 
-* `Use a distributed cluster with Dask-CUDA workers <Use Dask-CUDA>`_
-* `Use native cuDF spilling whenever possible <Enable cuDF Spilling>`_
+* Use a distributed cluster with `Dask-CUDA <https://docs.rapids.ai/api/dask-cuda/stable/>`__ workers
```

Comment:
I'm honestly not sure how to properly link to a different section within this page, so I'm just linking to the dask-cuda docs, etc. Happy to revise if others have advice.

Reply:
You can do inter-page links with syntax like this: https://www.sphinx-doc.org/en/master/usage/referencing.html#cross-referencing-syntax. No need to wait for this right now though.

Reply:
I just did that for this page, we can do that more broadly in `24.12`